sorts/merge_sort_fastest.py

```diff
@@ -4,16 +4,51 @@
 Best Case Scenario : O(n)
 Worst Case Scenario : O(n)
 '''
+# These modules are required for testing only.
+import random, copy
+import matplotlib.pyplot as plt
```

Comment:
Umm, it's nice to see the output as graph, it really helps in visualising the data. But I think it's not required here. We can just test for the output.

Reply:
@AnshulMalik 
good point! The function ```min``` and ```max``` have O(n). And this runtime is dominant.